.gitignore

```diff
@@ -57,3 +57,9 @@ wp-tests-config.php
 .svn
 
 vendor
+
+# IntelliJ project files
+.idea
+*.iml
+out
+gen
```

Comment:
There are hundreds of files that could be listed here for different IDEs, editors, etc.

Please remove this in favor of adding a **global** `.gitignore` file.  Here's how:  https://stackoverflow.com/a/7335487/106302

This also has the advantage that it works for all repositories on your computer, not just ClassicPress.